docs/yaml_output.md

```diff
@@ -0,0 +1,45 @@
+# YAML Format Output
+
+You can get the results in a YAML file to be consumed by a downstream tool or script.
+
+`$ tern -l report -f yaml -i golang:1.12-alpine -o output.yaml`
+
+A yaml foramt output file starts with line comments indicating the version of Tern. It has a `image` key(image.py) corresponding to the analyzed image. Here are the subkeys and their description:
+
+- `config`: the image config metadata
+- `image_id`: this is a unique identifier for the image - for OCI spec
+    this could be the digest. For now this is the sha256sum of the
+    config.json
+- `manifest`: the json object representing the image manifest
+- `layers`: list of layer objects in the image(image_layer.py)
+  - `diff_id`: the sha256 of the layer filesystem
+  - `fs_hash`: the hashed contents of the layer filesystem - default to empty string if there is no tarball of the layer filesystem
+  - `packages`: list of objects of type Package (package.py)
+    - `name`: package name
+    - `version`: package version
+    - `pkg_license`: package license that is declared
+    - `copyright`: copyright text
+    - `proj_url`: package source url
+    - `download_url`: package download url
+    - `origins`: a list of NoticeOrigin objects(notice_origin.py, expanded below)
+    - `checksum`: checksum as package property
+  - `origins`: list of NoticeOrigin objects (notice_origin.py, expended below)
+  - `tar_file`: the path to the layer filesystem tarball
+  - `created_by`: sometimes the metadata will contain a created_by
+      key containing the command that created the filesystem layer
+  - `import_image`: if the layer is imported from another image this is a pointer to that image. In Python terms it is just the name of the Image object or any object that uses this layer Based on how container image layers are created, this is usually the last layer of the image that was imported
```

Comment:
IIRC nothing shows up in the YAML here. It's probably a bug we need to fix.

Reply:
In the YAML output from the latest Release 1.0.1, I find theses keys under the `image`-`layers`-`analyzed_output`. For image with multiple layers, the analysis of each layer is under the key `analyzed_output`.